### DISCUSSION TITLE
Hi team, zkSync launch

### DISCUSSION THREAD

**Discussion by JoscelynFarr** [zkSync-Community-Hub/zksync-developers#50] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

Ser!! We hope to get in touch with zkSync marketing team as soon as possible because JOJO will publicly announce its launch on zkSync Era in 1 day!

### What is the specific issue or error you're encountering?

Can not connect the official Marketing.

### Can you share the error messages or logs you're receiving, if any?

nope

### Have you made any recent changes to the contract before encountering this issue?

nope

### Are there any external libraries or contracts that your contract interacts with?

nope

### Can you provide the relevant portions of your contract code where the issue is occurring?

nope

### Have you tried to isolate the problem, and if so, what were the results?

nope

### What steps have you already taken to try to resolve the issue?

Have already dm all marketing but no one response

**Comment by MexicanAce** [comment, marked as answer]
Hi @JoscelynFarr 👋 !

Please see the following page [Contact Us](https://zksync.io/contact) page on the zksync.io website for best ways to contact Media.

**Comment by bricnounlerumark** [comment]
It seems like you're experiencing difficulties reaching out to zkSync's marketing team. One suggestion would be to try reaching out through multiple channels, such as email, social media platforms like Twitter or LinkedIn, or even through the official zkSync website's contact form if available. Additionally, you could try joining relevant community forums or Discord channels where zkSync team members might be active. Persistence and exploring multiple communication channels might help in getting a response.

**Comment by whaliop** [comment]
Contacting zkSync marketing team for a launch announcement but unable to establish communication. Requesting urgent assistance.

**Comment by sfatitetyccindy** [comment]
Issue: Unable to connect with zkSync marketing team for a launch announcement.
Steps Taken: Attempted to contact marketing team via direct messages.
Status: No response received.

**Comment by latoyatidwell** [comment]
Contacting zkSync marketing team for a timely announcement of JOJO's launch on zkSync Era is crucial. However, unable to establish communication with the official marketing team despite multiple attempts.

**Comment by venlaudecsubs** [comment]
The user is trying to get in touch with the zkSync marketing team as soon as possible because they plan to publicly announce their launch on zkSync Era in one day. However, they are unable to connect with the official marketing team and have already tried reaching out to them via direct message with no response. No specific error messages or logs are provided, and no recent changes have been made to the contract. There are no external libraries or contracts that the contract interacts with. No relevant portions of the contract code are provided, and no steps have been taken to isolate the problem.
